include/seqan/sequence/string_base.h

```diff
@@ -1316,7 +1316,8 @@ struct AppendValueToString_
         TSize me_length = length(me);
         if (capacity(me) <= me_length)
         {
-            TTargetValue temp_copy(SEQAN_FORWARD(TValue, _value)); //temp copy because resize could invalidate _value
+            TTargetValue temp_copy;
+            assign(temp_copy, SEQAN_FORWARD(TValue, _value)); //temp copy because resize could invalidate _value
```

Comment:
std::string has no constructor for Segment<std::strng ...> but it does have an overloaded assign.